expat/configure.ac

```diff
@@ -253,8 +258,9 @@ AC_DEFINE([XML_DEV_URANDOM], 1,
 AC_ARG_ENABLE([xml-attr-info],
   [AS_HELP_STRING([--enable-xml-attr-info],
                  [Enable retrieving the byte offsets for attribute names and values
-                  @<:@default=no@:>@])], ,
-                 [enable_xml_attr_info=no])
+                  @<:@default=no@:>@])],
+    [],
```

Comment:
That's an argument to `AC_ARG_ENABLE`, not `AS_HELP_STRING`, right? Not matching with rule 2 of #390 yet then.